src/router/index.js

```diff
@@ -4,13 +4,18 @@ import routes from './routes';
 
 Vue.use(Router);
 
+const DISALLOW_SELECTOR_REGEX = /[;=/]/;
+
 export default function createRouter() {
 	return new Router({
 		mode: 'history',
 		routes,
 		scrollBehavior(to, from, savedPosition) {
-			if (to.hash) {
-				return { selector: to.hash };
+			if (to.hash && !DISALLOW_SELECTOR_REGEX.test(to.hash)) {
+				return {
+					selector: to.hash,
+					offset: savedPosition || { x: 0, y: 0 },
```

Comment:
I don't think we want this offset defined at all, as it's the offset relative to the position of the element represented by `selector` whereas `savedPosition` is the offset relative to the window

Reply:
Ah great catch